packages/@aws-cdk/aws-apigateway/lib/model.ts

```diff
@@ -1,46 +1,153 @@
-export interface IModel {
-    readonly modelId: string;
+import { Construct, IResource, Resource } from '@aws-cdk/core';
+import { CfnModel, CfnModelProps } from './apigateway.generated';
+import jsonSchema = require('./json-schema');
+import { IRestApi, RestApi } from './restapi';
+
+export interface IModel extends IResource {
+  /**
+   * A name for the model. If you don't specify a name, AWS CloudFormation
```

Comment:
These docs are suitable for the Props struct. This is the property that returns the attribute, so it will always have a value (even if it’s a token)

Reply:
Done